presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -58,12 +61,13 @@ public void setup()
                 "\"col_0\" BIGINT, " +
                 "\"col_1\" DOUBLE, " +
                 "\"col_2\" BOOLEAN, " +
+                "\"col_3\" VARCHAR(128), " +
                 ")");
         preparedStatement.execute();
         StringBuilder stringBuilder = new StringBuilder("insert into \"test_table\" values ");
         int len = 1000;
         for (int i = 0; i < len; i++) {
-            stringBuilder.append("(" + i + ", " + (200000.0 + i / 2.0) + ", " + (i % 2 == 0) + ")");
+            stringBuilder.append("(" + i + ", " + (200000.0 + i / 2.0) + ", " + (i % 2 == 0) + ", 'test_str_" + i + "')");
```

Comment:
This is really hard to read. Extract the expressions into variables and use `format()` to create the string